base_report_auto_create_qweb/__init__.py

```diff
@@ -1,7 +1,7 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-# For copyright and license notices, see __openerp__.py file in root directory
-##############################################################################
+# -*- coding: utf-8 -*-
+# Authors: See README.RST
+# Copyright 2016 See README.rst for Authors
```

Comment:
BTW Authors and copyrights are not necessary the same. Example: I could be an author, but the copyright would be granted to Camptocamp.